coderkevin/fresh-data
# Add WP REST API example

This adds the boilerplate code from create-react-app and then modified for looks and simplicity for the example app. No actual fresh-data code here yet.

To Test:
1. `cd examples/wp-rest-api`
2. `npm install`
3. `npm test` and ensure the 1 test passes.
4. `npm start` and ensure that the page comes up.
5. `Click the WordPress REST API` title and it should go to the WP REST API docs.
